packages/compile-solidity/src/compilerSupplier/loadingStrategies/VersionRange.ts

```diff
@@ -6,7 +6,19 @@ import originalRequire from "original-require";
 
 // must polyfill AbortController to use axios >=0.20.0, <=0.27.2 on node <= v14.x
 import "../../polyfill";
-import { default as axios, AxiosResponse } from "axios";
+import { default as axiosPlain, AxiosResponse } from "axios";
+import { HttpsProxyAgent } from "https-proxy-agent";
+
+const environmentProxy = process.env.https_proxy || process.env.http_proxy;
+let axios;
+if (environmentProxy) {
+  const agent = new HttpsProxyAgent(environmentProxy);
+  axios = axiosPlain.create({
+    httpsAgent: agent
+  });
+} else {
+  axios = axiosPlain;
+}
```

Comment:
I think the most recent rebase undid initializing `axios` to `axiosPlain`

Reply:
Yes I will be able to test with this release.

Reply:
Hi @joseluu, did the latest release resolve your proxy issue?  I don't have an easy way to test this :/ I wonder if it's related to [this `TunnelAgent.httpsOverHttp` comment](https://github.com/axios/axios/issues/3384#issuecomment-1228879716) in a related Axios issue. 🤔

Reply:
Thanks @joseluu! Bummer the dep upgrade didn't resolve the issue. You mentioned there were problems with axios 1.2.4 above, can you confirm your PR still works for your configuration?

This issue has been dogging us for a while, and I'd like to document the various proxy configurations, so we can incorporate proxy testing in CI. Would you be able to describe your setup? Thanks again!

Reply:
I rebased the patch on v5.7.4, and checked operation behind a firewall: it works